data/blog/electron-12-0.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Electron 12.0.0
+author:
+- VerteDinde
+- mlaurencin
+- sofianguy
+date: '2021-03-02'
+---
+
+Electron 12.0.0 has been released! It includes upgrades to Chromium `xx`, V8 `xx`, and Node.js `xx`. We've added (features), and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 12.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `xx`
+    * [New in Chrome 88]()
+    * [New in Chrome 89]()
+* Node.js `xx`
+    * [Node 14.15.1 blog post]()
+* V8 `8.x`
+    * [V8 8.4 blog post]()
```

Comment:
V8.8 blog doesn't appear on their site 🤔 
So we can remove this line and just list the 8.9 blog: https://v8.dev/blog/v8-release-89